daemon/oci_windows.go

```diff
@@ -25,11 +25,30 @@ func (daemon *Daemon) createSpec(c *container.Container) (*specs.Spec, error) {
 	// In base spec
 	s.Hostname = c.FullHostname()
 
+	if err := daemon.setupConfigDir(c); err != nil {
+		return nil, err
+	}
+
 	// In s.Mounts
 	mounts, err := daemon.setupMounts(c)
 	if err != nil {
 		return nil, err
 	}
+
+	if len(c.ConfigReferences) > 0 {
+		if err := daemon.Mount(c); err != nil {
```

Comment:
Per slack, but putting on GH too. `createSpec` on Windows is called for starting a  container. For Hyper-V containers, it's not safe to mount the sandbox on the host if the container has been started before as it's an attack vector on the host.